components/inputs/test/input-date-time-range.visual-diff.js

```diff
@@ -223,8 +223,8 @@ describe('d2l-input-date-time-range', () => {
 				describe(testCase.name, () => {
 					before(async() => {
 						await page.$eval('#min-max', (elem) => elem.blur());
-						await changeInnerInputTextDate(page, '#min-max', startDateSelector, testCase.startDate);
-						await changeInnerInputTextDate(page, '#min-max', endDateSelector, testCase.endDate);
+						await changeInnerInputDateTime(page, '#min-max', startDateSelector, testCase.startDate);
```

Comment:
This test was to trigger error handling but since the times were no longer the same (changed to startOfDay and endOfDay) there was no longer an error so the time has to be tweaked to be the same as well